### ISSUE TITLE
Add resource server commands

### ISSUE THREAD

**Issue by Disinterpreter** [multitheftauto/mtasa-blue#2549]
**Is your feature request related to a problem? Please describe.**
Add serverside commands like in FiveM 

**Describe the solution you'd like**
Improve command `start\restart\stop` and add `ensure`

`start\restart\stop`  must support directories with brackets like
`start [cars]`
`ensure` restart the resource (or group) and start if something in argument was stopped like
`ensure [cars]`
`ensure resourcename`

**Additional context**
https://docs.fivem.net/docs/server-manual/server-commands/

P.S perhaps it is *good first issue*

**Comment by Fernando-A-Rocha** [comment]
ensure seems useful! +1

**Comment by 0xCiBeR** [comment]
What would be the point of supporting bracket type directories? The way it currently works is that if a directory has [] in its name its ignored as a resource since its used for having better sorting inside the resource folder. Adding this would break that functionality.
The ensure command would be great.

**Comment by Lpsd** [comment]
Adding to that, we already have the `require` tag in [meta.xml](https://wiki.multitheftauto.com/wiki/Meta.xml) if you want to start multiple resources.

**Comment by CrosRoad95** [comment]
no, in my opinion we should add server config that says which commands should not be added by default, for example "start" and then you can implement this functionality by yourself

**Comment by PlatinMTA** [comment]
> What would be the point of supporting bracket type directories?

I think he wants to start, stop or restart all the resources inside a [directory].
